packages/amplify-app/src/call-amplify.js

```diff
@@ -6,14 +6,14 @@ const amplify = isWin ? 'amplify.cmd' : 'amplify';
 const amplifyDev = isWin ? 'amplify-dev.cmd' : 'amplify-dev';
 
 const defaultOpts = {
-  inheritIO: true,
+  inheritIO: true
```

Comment:
Is the linter removing trailing commas? I like them because they simplify git diffs when entries change.

Reply:
I'm a fellow fan of trailing commas. I added a prettierrc with that configuration.